### DISCUSSION TITLE
Increase private group size to 1000 individuals

### DISCUSSION THREAD

**Discussion by matthewMGMR** [oxen-io/session-desktop#2606] [Ideas]
Since it was announced that Amazon's Wickr Me is shutting down people are looking for new privacy centered alternatives but there is one thing holding individuals back from joining Session: the limited size of private groups (100).

Wickr Me had a max group size of 500 which allowed huge communities and teams to communicate together so Session should have a limit of at least 500, preferably 1000. 

Would this be possible to implement in the near future?

Best Regards, Matthew

**Comment by KeeJef** [comment]
We want to raise the limit, but we need to fix closed groups first, right now they are very unstable. In the future if we raise the limit we also need to think about how that scales, especially around kicking members in large groups since required key rotations which can be very resource intensive on group administrators
